README.md

```diff
@@ -32,7 +32,7 @@ $ tsd query jquery.*
 $ tsd query "jquery.*"
 
 #install all definitions from tsd.json
-$ tsd install
+$ tsd reinstall
```

Comment:
Nice! Here (https://github.com/DefinitelyTyped/tsd/releases) you can see the latest published versions :)